### ISSUE TITLE
Unicode Fehler bei nicht-ASCII Zeichen in Feld "Bearbeitungsinformation"

### ISSUE THREAD

**Issue by lukasgraf** [4teamwork/opengever.core#6204]
_Harvest: `[3.2.2.1] OneGov GEVER` - `Softwarepflege (Refactoring/Bug)`_

Nicht-ASCII Zeichen im Feld "Bearbeitungsinformation" (`public_trial_statement `) führen zu einem Unicode Fehler beim Darstellen des Dokuments.

### Steps to reproduce

- Das im Sentry-Issue angegebene Dokument öffnen

https://sentry.4teamwork.ch/sentry/onegov-gever/issues/58646/

Siehe auch: https://basecamp.com/2768704/projects/14549453/todos/408873015

**Comment by njohner** [comment]
It seems the field is saved as unicode on objects older than 2016, exact date being hard to determine. This has nothing to do with the version bump, as it is already broken on the previous versions. See for example ska-arch/ordnungssystem2/leitung/dossier-2158/dossier-2159/document-8631 on zug PROD which still runs `2019.2.6`.

Proposed solution is an upgrade step to fix the older objects. This should be fairly cheap as it only affects older objects, of which there are rather few.

**Comment by njohner** [comment]
newest object with wrong field type dates from 2015/07/24 and the first one with correct type is dates from 2015/07/28. So it seems it was fixed with update from GEVER 3.1 to GEVER 3.4: https://github.com/4teamwork/opengever.zug/pull/181

Here a commit that hints towards encoding issue with the public_trial_statement:
https://github.com/4teamwork/opengever.core/pull/473/files

**Comment by njohner** [comment]
Fixed the one object they reported with:
```
from opengever.maintenance import dm; dm()
from plone import api
portal = api.portal.get()
from zope.component.hooks import setSite; setSite(portal)

obj_url = "ska-arch/ordnungssystem2/oeffentlichkeit/allgemeines-und-uebergreifendes/dossier-148/document-409"
obj = portal.unrestrictedTraverse(obj_url)
obj.public_trial_statement = obj.public_trial_statement.encode('utf-8')
```

**Comment by njohner** [comment]
PR with proposed upgrade step to fix this for Zug

**Comment by lukasgraf** [comment]
Upgrade-Step needs backport to `2019.6-stable` branch.

**Comment by njohner** [comment]
Backported and released `2019.6.2`
